beacon-chain/rpc/node/server.go

```diff
@@ -47,9 +47,14 @@ func (ns *Server) GetGenesis(ctx context.Context, _ *ptypes.Empty) (*ethpb.Genes
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not convert genesis time to proto: %v", err)
 	}
+	genState, err := ns.BeaconDB.GenesisState(ctx)
```

Comment:
Not sure how else to get the genesis state from what `Server` has access to, getting the whole genesis state seems expensive. Maybe we can add GenesisValRoot to the DB like the deposit contract?